README.md

```diff
@@ -158,8 +159,8 @@ Here you can see the details about various variables in configuration
 - CTP_AUTH_URL: The URL for authentication in commercetools platform. It is used to generate OAuth 2.0 token which is required in every API call to commercetools composable commerce. The default value is `https://auth.europe-west1.gcp.commercetools.com`. For details, please refer to documentation [here](https://docs.commercetools.com/tutorials/api-tutorial#authentication).
 - CTP_API_URL: The URL for commercetools composable commerce API. Default value is `https://api.europe-west1.gcp.commercetools.com`.
 - CTP_SESSION_URL: The URL for session creation in commercetools platform. Connectors relies on the session created to be able to share information between enabler and processor. The default value is `https://session.europe-west1.gcp.commercetools.com`.
-- CTP_JWKS_URL: The URL which provides JSON Web Key Set.
-- CTP_JWT_ISSUER: The issuer inside JSON Web Token which is required in JWT validation process.
+- CTP_JWKS_URL: The URL which provides JSON Web Key Set. Default value is `https://mc-api.europe-west1.gcp.commercetools.com`. 
```

Comment:
```suggestion
- CTP_JWKS_URL: The URL which provides JSON Web Key Set. Default value is `https://mc-api.europe-west1.gcp.commercetools.com/.well-known/jwks.json`. 
```

It should ge the following:

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/103/commits/879d9a079bfdcf944f6dba5b7b4741027284d615